symengine/tests/printing/test_printing.cpp

```diff
@@ -42,6 +42,12 @@ using SymEngine::complex_double;
 using SymEngine::BaseVisitor;
 using SymEngine::StrPrinter;
 using SymEngine::Sin;
+using SymEngine::integer_class;
+
+inline integer_class operator "" _z(const char* str)
+{
+    return integer_class(str);
+}
```

Comment:
Actually, I can see that you did that already, but then moved it into the `.cpp` files in 79bde6db22a746ba5fdee88416c57d15bfff1d73, but the commit log does not explain why.